sysid-library/src/main/include/sysid/generation/SysIdSetup.h

```diff
@@ -77,13 +78,15 @@ void SetMotorControllers(
  * @param rate A reference to a function that is supposed to return the encoder
  *             rate
  */
-void SetupEncoders(std::string_view encoderType, bool isEncoding, int period,
-                   double cpr, int numSamples, std::string_view controllerName,
-                   frc::MotorController* controller, bool encoderInverted,
-                   const std::vector<int>& encoderPorts,
-                   // std::unique_ptr<CANCoder>& cancoder,
-                   std::unique_ptr<frc::Encoder>& encoder,
-                   std::function<double()>& position,
-                   std::function<double()>& rate);
+void SetupEncoders(
+    std::string_view encoderType, bool isEncoding, int period, double cpr,
+    int numSamples, std::string_view controllerName,
+    frc::MotorController* controller, bool encoderInverted,
+    const std::vector<int>& encoderPorts,
+    // std::unique_ptr<CANCoder>& cancoder,
+    std::unique_ptr<rev::SparkMaxRelativeEncoder>& revEncoderPort,
+    std::unique_ptr<rev::SparkMaxAlternateEncoder>& revDataPort,
```

Comment:
The non-owning version of `std::unique_ptr<T>` is `T*`, not `std::unique_ptr<T>&`.

Reply:
Nevermind. This is an out parameter, not an in parameter.